WeatherToday/Sources/Domain/Entities/Weather.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Weather.swift
+//  WeatherToday
+//
+//  Created by Taher on 7/10/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+protocol Weather {
+    var temperature: Double { get }
+    var humidity: Double { get }
+}
+
+
+struct WeatherModel: Decodable {
+    let main: WeatherN
+}
+
+struct WeatherN: Decodable {
```

Comment:
```suggestion
struct WeatherN: Decodable {

```

Reply:
updated in this [commit](https://github.com/nimblehq/ci-cd-sample-ios/pull/20/commits/0df0b8e6ef1945dbb46c9bd130e52d369b861265)